web-ui/src/components/personnel/Personnel.jsx

```diff
@@ -69,7 +68,7 @@ const Personnel = () => {
   }, [csrf, personnel, dispatch]);
 
 // Create feedback request link
-const createFeedbackRequestLink = (memberId) => (<Link to={`/feedback/request?for=${memberId}`}>Request Feedback</Link>);
+const createFeedbackRequestLink = (memberId) => (<Link className="feedback-link" to={`/feedback/request?for=${memberId}&step=1`}>Request Feedback</Link>);
```

Comment:
Slight change: link to `/feedback/request/?for=etc` (adding a slash in front of the question mark) This does not change anything functionality-wise but keeps it consistent with our other URLs